static-assets/components/cstudio-admin/mods/admin-configurations.js

```diff
@@ -175,12 +175,12 @@
           /**/ `<h2 class="hint--title">${formatMessage(adminConfigurationMessages.encryptMarked)}</h2>` +
           /**/ `<p>${formatMessage(adminConfigurationMessages.encryptHintPt1)}</p>` +
           /**/ `<p>` +
-          /**/ formatMessage(adminConfigurationMessages.encryptHintPt2, bold).join('') +
+          /**/ formatMessage(adminConfigurationMessages.encryptHintPt2, bold).props.children.join(' ') +
```

Comment:
Think this is a dangerous. I've been holding updating react-intl for this reason. Let's discuss.

Reply:
Reverted react-intl version